src/scss/_spark-theme.scss

```diff
@@ -49,4 +49,22 @@ body {
   visibility: hidden;
 }
 
+.docs-c-SkipNav {
+  position: absolute;
+  top: 0;
+  z-index: 5000;
+
+  a {
```

Comment:
We try not to directly select HTML elements. Could you update this? Maybe docs-c-SkipNav__link ?